src/components/ViewTeams/container.js

```diff
@@ -36,12 +66,14 @@ export default Wrapped =>
     };
 
     render() {
+      const { clients, teams, selectedTeam } = this.state;
       return (
         <Wrapped
-          history={this.props.history}
+          navigateTo={this.props.history.push}
```

Comment:
I'm not so sure about this mainly because it's breaking away from what is done everywhere else in the app.